libs/librtlnumber/src/rtl_int.cpp

```diff
@@ -324,6 +348,28 @@ VNumber V_BITWISE_XNOR(VNumber& a)
  *                                                                          
  */
 
+VNumber V_REPLICATE(VNumber& a, VNumber& n_times)
+{
+	return a.replicate(n_times);
+}
+
+VNumber V_CONCAT(std::vector<VNumber> concat_list)
+{
+	if(concat_list.empty())
+	{
+		return VNumber("0"); // todo: what to do?
```

Comment:
The standard doesn't explicitly state what to do with empty concats, but this would be equivalent to ```a = + ;``` - an operation without operands. 

The bison does require operands for any expression, including within the brackets for concats, so empty concats should never reach this point. If it somehow did though, it should be an error.

Reply:
fixed